sigar_interface.go

```diff
@@ -167,3 +169,16 @@ func (self DiskIo) Delta(other DiskIo) DiskIo {
 		IoTimeMs: self.IoTimeMs - other.IoTimeMs,
 	}
 }
+
+type SystemInfo struct {
```

Comment:
Possibly. I envisioned it going something like this:

```
Sysname: Windows
Nodename: hostname
Release: Possibly version of `ntoskrnl.exe`
Version: Undecided
Machine: Not sure the best API to get this, its probably x64
Domainname: blank
```

I know its weird, but Windows seems to fit into at least a subset of the `uname`. Is there other info you think we need?

Also, I was planning to put the Windows version (e.g. `Windows XP`) in `SystemDistribution.Description`. Do you have other suggestions for better organizing?